### PR TITLE
Change istio 0.7 to 0.8

### PR DESCRIPTION
**Pull request by aljesusg**
Require [kiali #282](https://github.com/kiali/kiali/pull/282)

cc @pilhuhn

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -108,11 +108,11 @@ If you do not already have an Istio-enabled cluster but want to see what Kiali i
 
 ```
 wget https://raw.githubusercontent.com/kiali/kiali/master/hack/cluster-openshift.sh
-sh cluster-openshift.sh --istio-version 0.7.1 --kiali-enabled true up
+sh cluster-openshift.sh --istio-version 0.8 --kiali-enabled true up
```
Comment (jmazzitelli) [inline index.md]: I don't think this is right. I believe you need to specify an exact, specific version - 0.8.0 for example.

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

File: index.md [hunk 2]
```diff
@@ -108,11 +108,11 @@ If you do not already have an Istio-enabled cluster but want to see what Kiali i
...
 ```
 
 [NOTE]
-Notice the above command line forces the version of Istio to 0.7.1. Kiali will support Istio 0.8.0 in the very near future, but for now, we recommend using --istio-version 0.7.1 if you just want to see what Kiali is all about.
+Notice the above command line forces the version of Istio to 0.8 Kiali will support Istio 0.8.0 in the very near future, but for now, we recommend using --istio-version 0.8 if you just want to see what Kiali is all about.
```
Comment (jmazzitelli) [inline index.md]: This comment now doesn't make sense. I think we should either remove this note (lines 114-5) completely or change it to say something like "we are recommending 0.8.0 in the above command, but keep in mind Kiali is currently in active development to get it to support 0.8.0 fully. You could try 0.7.1 instead."

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

File: index.md [hunk 3]
```diff
@@ -108,11 +108,11 @@ If you do not already have an Istio-enabled cluster but want to see what Kiali i
 
 ```
 wget https://raw.githubusercontent.com/kiali/kiali/master/hack/cluster-openshift.sh
-sh cluster-openshift.sh --istio-version 0.7.1 --kiali-enabled true up
+sh cluster-openshift.sh --istio-version 0.8.0 --kiali-enabled true up
```
Comment (jmazzitelli) [inline index.md]: --istio-version doesn't need to be specified now - 0.8.0 is the latest and is the default.

File: index.md [hunk 4]
```diff
@@ -108,11 +108,11 @@ If you do not already have an Istio-enabled cluster but want to see what Kiali i
...
 ```
 
 [NOTE]
-Notice the above command line forces the version of Istio to 0.7.1. Kiali will support Istio 0.8.0 in the very near future, but for now, we recommend using --istio-version 0.7.1 if you just want to see what Kiali is all about.
+We are recommending 0.8.0 in the above command, but keep in mind Kiali is currently in active development to get it to support 0.8.0 fully. You could try 0.7.1 instead..
```
Comment (jmazzitelli) [inline index.md]: delete this line entirely

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

**Comment by aljesusg** [comment]
@jmazzitelli I did the changes

**Comment by pilhuhn** [comment]
Looks good now

**Pull request merged**